charts/aws-ebs-csi-driver/templates/attacher.yaml

```diff
@@ -0,0 +1,166 @@
+apiVersion: v1
```

Comment:
we combined attacher and provisioner manifest few weeks back. Could you combine helm template as well? 

There are a bunches of changes made along with the combine. I think its easier to create new helm templates referring the existing manifests